OracleWebLogic/samples/12212-msiserver/README.md

```diff
@@ -94,35 +87,18 @@ So "simple_filename" helps identify the name of the file where the source is cop
 Using swarm service creation with this image
 --------------------------------------------
 The image called 12212-summercamps-msiserver can be used for Docker service creation
-to scale out to multiple replicas. docker service creation can only be done
-using an image from a docker registry. If you don't have publishing rights to a docker
+to scale out to multiple replicas. Docker service creation can only be done

```

Comment:
This is not accurate: Docker service creation can be done either using a local registry or by ensuring the same image is available to all hosts. Using a local registry is just easier, but it's not mandatory.